webapi/src/main/scala/org/knora/webapi/routing/RouteUtilV2.scala

```diff
@@ -153,7 +153,7 @@ object RouteUtilV2 {
     /**
       * Sends a message to a responder and completes the HTTP request by returning the response as TEI/XML.
       *
-      * @param requestMessage   a future containing a [[KnoraRequestV2]] message that should be sent to the responder manager.
+      * @param requestMessageF  a future containing a [[KnoraRequestV2]] message that should be sent to the responder manager.
```

Comment:
Since `runRdfRoute` is now called only by `runRdfRouteWithFuture`, you could make `runRdfRoute` private.

Reply:
done